atomicapp/nulecule/main.py

```diff
@@ -228,6 +228,10 @@ def run(self, cli_provider, answers_output, ask,
         if not self.answers_file:
             self._process_answers()
 
+        # Give more prefernce to provider, given on cmd-line
```

Comment:
What here I mean is that somebody can give provider in two ways simultaneously! Via answers file and via CLI.